AutomationHelpers/src/UserCodeCollections/StringLibrary.cs

```diff
@@ -24,5 +24,29 @@ public static string ConcatStrings(string value1, string value2)
         {
             return String.Concat(value1, value2);
         }
+
+        /// <summary>
+        /// Creates and returns a random string. If a string shorter than 35 characters is used it may not be unique.
+        /// </summary>
+        /// <param name="length">Expected length of the return value (default length is 35)</param>

```

Comment:
Length of the returned string in characters. Default = 35. Lower values increase chance of duplicate strings.